### ISSUE TITLE
EasyMotion commands as an argument for 'd' or 'c' don't work anymore

### ISSUE THREAD

**Issue by meseck** [AlexPl292/IdeaVim-EasyMotion#56]
For me, it only selects the area between the last cursor position and the jump target.

![](https://user-images.githubusercontent.com/34334994/116442254-92e1fa80-a852-11eb-8d76-04214110d3fc.gif)

I've already read that the new AceJump release has broken some things. Is this maybe on of the things?

**Comment by AlexPl292** [comment]
Hi! With the new releases of AceJump (3.7.1) and IdeaVim-EasyMotion (1.6) everything should work correct. Could you please specify what keys do you press on the gif?

**Comment by fools-mate** [comment]
Hello, thank you for the quick reply!

I used 'd\<leader\>j' in this case and this is mapped to '<plug>(easymotion-j)'. In nvim it works just fine with this configuration:

```
" Search for character forwards and put the cursor on the character
map <Leader>f <Plug>(easymotion-f)
map <Leader>F <Plug>(easymotion-F)

" Search for character forwards and put the cursor before the character
map <Leader>t <Plug>(easymotion-t)
map <Leader>T <Plug>(easymotion-T)

" Left(H)
map <Leader>h <Plug>(easymotion-b)
map <Leader><Leader>h <Plug>(easymotion-ge)

" Down(J)
map <Leader>j <Plug>(easymotion-j)

" Up(K)
map <Leader>k <Plug>(easymotion-k)

" Right(L)
map <Leader>l <Plug>(easymotion-w)
map <Leader><Leader>l <Plug>(easymotion-e)
```

**Comment by brunuh** [comment]
I'm experiencing the same issue, and have AceJump 3.7.1 and IdeaVim-EasyMotion 1.6 as well.

I'm mostly using: `d<leader><leader>w` (followed by the easymotion shortcut), and does nothing. It does leave the `d` command left behind as any follow up motion removes whatever that motion includes.

Worth noting that I've noticed this failing for a couple of weeks by now.

**Comment by rbhanot4739** [comment]
+1 same issue for me as well. Using the latest version of IntelliJ, IdeaVim, AceJump and IdeaVim-EasyMotion

**Comment by rbhanot4739** [comment]
@AlexPl292 Can you please let us know when can this be targetted for a fix, this is sort of a deal breaker when using easy-motion. Easy-motion feels way too ineffective without this bug fix.

**Comment by zhaohongxuan** [comment]
same issue now

**Comment by dayoffrettchen** [comment]
same issue. Looked into sneak but the . command is broken.

**Comment by pindab0ter** [comment]
Same issue here.

**Comment by afezeria** [comment]
same issue now.
AceJump (3.8.5) and IdeaVim-EasyMotion (1.6)

**Comment by krishanunayak** [comment]
5 months later, still broken! :(

**Comment by AlexPl292** [comment]
Hi folks, this should be fixed in 1.8 release (with IdeaVim 1.10.+).
